xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-theme/xwiki-platform-flamingo-theme-bootswatch/src/main/resources/FlamingoThemes/Cyborg.xml

```diff
@@ -2050,6 +2063,9 @@ a.list-group-item {
     <property>
       <panel-bg>#060606</panel-bg>
     </property>
+    <property>
+      <panel-default-text>#ffffff</panel-default-text>
```

Comment:
This looks like a bad merge.

Reply:
Addressed in https://github.com/xwiki/xwiki-platform/pull/2497/commits/9c640c2eaae7adb96e7a584c41293f63e6e580b0 :+1: